app/models/middleware_server.rb

```diff
@@ -42,11 +42,11 @@ def in_domain?
 
   # Returns whether a server is immutable or not.
   #
-  # By default, we define all servers as being immutable, so that power
-  # operations are not allowed, and it is the provider responsability to
+  # By default, we define all servers as being mutable, so that power
+  # operations are allowed, and it is the provider responsability to
   # reimplement this method if necessary.
   def immutable?
-    true
+    false
```

Comment:
It is overwritten here: https://github.com/ManageIQ/manageiq-providers-hawkular/blob/master/app/models/manageiq/providers/hawkular/middleware_manager/middleware_server.rb

As of now I'm tracking what might be the error where this class is not being instantiated on miq-core, so this change would only change the default if the provider does not have a different implementation.